content/docs/atom-guide/look-dev/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,669 @@
+7. Open the **Editor** and you should see your tree's leaves bending slightly. If you don't, try opening the **Material Editor** and increasing all the parameters. 
+
+{{< video src="/images/atom-guide/vegetation-bending-tutorial/detailbendingtree.mp4" autoplay="true" loop="true" width="100%" muted="true" info="Video of tree detail bending." >}}
+
+### Add main bending
+The leaves are moving now, but the tree doesn't sway yet. We will now add main bending, the overall sway and movement that the whole tree experiences.
+
+1. Open `o3de\Gems\Atom\Feature\Common\Assets\Materials\Types\VegetationBending_Common.azsli`.
+2. At the bottom, add the following code:
+   
+   ```
+   float3 mainBending(float3 position, float4 bending) {
+      float bendFactor = position.z * bending.z;
+      bendFactor *= bendFactor; 
+
+      float len = length(position); 
+
+      float3 newPos = position;
+      newPos.xy += float2(bending.x, bending.y) * bendFactor;
+
+      return normalize(newPos) * len;
+   }
+   ```
+   Using the current position of the vertex (after it has been changed from the detail bending) and the bending determined by the wind, we can bend the tree as a whole.
+3. Call the detail bending function in the vertex shaders. Open `o3de\Gems\Atom\Feature\Common\Assets\Materials\Types\VegetationBending_ForwardPass.azsli`.
+4. Find the vertex shader (`VegetationBending_ForwardPassVS`) and add a call to our detail bending function under the wind function call:
+   
+   ```
+   if (o_color_isBound) {
+      float currentTime = SceneSrg::m_time;
+
+      // Overall wind
+      float4 currentBending = setUpWindBending(currentTime, worldPosition);
+
+      // Detail bending
+      float3 currentOutPosition = detailBending(IN.m_position, IN.m_normal, IN.m_optional_color, currentTime, worldPosition, currentBending.w);
+
+      currentOutPosition = mainBending(currentOutPosition, currentBending);
+
+      worldPosition = mul(objectToWorld, float4(currentOutPosition, 1.0));
+   }
+   ```
+5. Repeat steps 3-4 with the depth pass in `o3de\Gems\Atom\Feature\Common\Assets\Materials\Types\VegetationBending_DepthPass.azsli` and the depth pass with PS in `o3de\Gems\Atom\Feature\Common\Assets\Materials\Types\VegetationBending_DepthPass_WithPS.azsli`.
+6. Open the **Editor** and you should see your tree swaying with the leaves still bending. If you don't, try opening the **Material Editor** and increasing the wind parameters.
+
+Amazing, our tree now sways and reacts to wind! Try to place multiple trees and observe how the trees sway differently when close together versus farther away. Also, add some lighting to make the trees pop!
+
+{{< video src="/images/atom-guide/vegetation-bending-tutorial/tree3.mp4" autoplay="true" loop="true" width="100%" muted="true" info="Video of trees swaying." >}}
+
+### Add motion vectors
+Because our tree is moving, we can add a *motion vector pass* so we can choose to add motion blur or other effects in the future.
```

Comment:
We do have a partially implemented Temporal Anti-Aliasing (TAA) feature that can utilize the motion vectors. It isn't complete but mentioning this might be better than just saying we're adding motion vectors just in case it's used later. You might even be able to turn on TAA, demonstrate that it doesn't look right when the tree is moving (there should be obvious artifacts), and then you can hopefully show an improvement after you add the wind bending shader.